egs/wsj/s5/steps/nnet3/tdnn/train.sh

```diff
@@ -5,7 +5,7 @@
 # Copyright 2012-2015  Johns Hopkins University (Author: Daniel Povey).
 #           2013  Xiaohui Zhang
 #           2013  Guoguo Chen
-#           2014  Vimal Manohar
+#           2014-2016  Vimal Manohar
```

Comment:
BTW, you might find it easier to have the top-level train.sh script not be responsible for generating the configs or the egs-- if you assume that the top-level run.sh calls the appropriate scripts beforehand, that would make your life easier.  I'm following this pattern in the xvector setup.